uk.co.compucorp.civicrm.hrcore/backstop_data/page-objects/tabs/absence/workpatterns.js

```diff
@@ -0,0 +1,14 @@
+var page = require('./../../page');
+
+module.exports = (function () {
+  return page.extend({
+    /**
+     * Wait for the page to be ready
+     */
+    waitForReady: function () {
+      var casper = this.casper;
+      casper.click('[heading="Work Patterns"] > a');
```

Comment:
This doesn't look reliable to me, it will stop working if we just change text, which is common. I think it is always better to select by either classes or additional attributes. Please take a look how it is done here https://github.com/civicrm/civihr/pull/1952/files#diff-44f7e27aca0c60ef58ab45ba88811751